README.md

```diff
@@ -67,7 +77,12 @@ The default settings are to wait until the deployment process is finished and th
           "host": 80,
           "container": 80
         }
-      ]
+      ],
+      "mountPoints": {
```

Comment:
```suggestion
      "mounts": {
```

Normalerweise bezeichnet man mit "Mountpoint" nur das Zielverzeichnis, in das man reinmountet.

Reply:
Ich habe mich an diesen Stellen bemüht, die Config so nah wie möglich an der von AWS verwendeten Config zu halten, und da heißt das Konstrukt `mountPoints`. Das betrifft auch die Struktur der Volumes.

Reply:
Ich hätte gerne bei uns überall entweder "xxx_yyy" oder "xxxYyy" gesehen. Bitte in `xxx_yyy` umändern.

Reply:
Ich finde auch `mounts` besser - siehe hier z.B. - https://docs.docker.com/storage/bind-mounts/